From: Colin Ian King <colin.i.king@gmail.com>
Subject: selftests/mm: Fix spelling mistake "priviledge" -> "privilege"
Date: Fri, 14 Apr 2023 09:15:06 +0100

There is a spelling mistake in an error message string. Fix it.

Link: https://lkml.kernel.org/r/20230414081506.1678998-1-colin.i.king@gmail.com
Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
Cc: Peter Xu <peterx@redhat.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/testing/selftests/mm/uffd-common.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/tools/testing/selftests/mm/uffd-common.c~selftests-mm-allow-uffd-test-to-skip-properly-with-no-privilege-fix
+++ a/tools/testing/selftests/mm/uffd-common.c
@@ -314,7 +314,7 @@ int uffd_test_ctx_init(uint64_t features
 	ret = userfaultfd_open(&features);
 	if (ret) {
 		if (errmsg)
-			*errmsg = "possible lack of priviledge";
+			*errmsg = "possible lack of privilege";
 		return ret;
 	}
 
_
